.github/workflows/azure-podvm-image-build.yml

```diff
@@ -49,6 +49,7 @@ jobs:
         ref: "${{ inputs.git-ref || 'main' }}"
 
     - name: Read properties from versions.yaml
+      working-directory: cloud-api-adaptor
```

Comment:
Does it supersede the global `working-directory: cloud-api-adaptor/azure/image`  ? Or the working-directory here should be like `../../`?

Reply:
I think it still cannot find the `versions.yaml`: https://github.com/confidential-containers/cloud-api-adaptor/issues/1446#issuecomment-1728129388